requirements.txt

```diff
@@ -1,8 +1,9 @@
-django==1.8
+django<1.9
```

Comment:
Hmm... Good point, thanks. How do you deal with keeping stuff up to date? Wait till you need it, or run regular upgrades?
This was triggered by a bug in 1.8.0 (I meant to use the newest 1.8 when writing it initially, but 1.8 means 1.8.0)